docs/DeveloperGuide.md

```diff
@@ -271,33 +271,95 @@ This feature has been enhanced based on the initial implementation of the `edit`
 
 #### How is this feature implemented?
 
-The proposed edit mechanism is facilitated by `EditCommand`. It is supported by `EditCommandParser` where it will retrieve the attributes that can be edited.
+The proposed edit mechanism is facilitated by `EditCommand`. It is supported by `EditCommandParser` and `EditPersonDescriptor` 
+where it will retrieve the attributes that can be edited.
 
 The user is able to edit key attributes of the Candidate, such as
 * Phone Number
 * ApplicationStatus
-* InterviewStatus
 
 Based on the fields that are edited by the user, the EditCommandParser will retreive the information, and update the attributes accordingly.
-There are also some attributes that are dependent on another attribute, and we will introduce `Triggers` to update the attributes automatically.
-One of such is the `InterviewStatus`. It makes sense that if the application is accepted or rejected, it also means that the interview has been completed.
-Hence the `ApplicationStatus` will trigger `InterviewStatus` if the user is changing the `ApplicationStatus` from `Pending` to `Accepted` or `Rejected`.
-The `trigger mechanism` (WIP) is currently handled by `EditCommand#triggerInterviewStatus()`.
+This `EditPersonDescriptor` class will take in the current attributes of the Candidate, and update the fields accordingly by the
+`EditCommandParser`. 
```

Comment:
This too. Just curious!

Reply:
eh my bad ! I think i accidentally pushed to the next line on intellij!